<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1024</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1024</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="settingBtn">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>10</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>ပြင်ဆင်ခြင်း</string>
    </property>
   </widget>
   <widget class="QPushButton" name="menuBtn">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>ဆော့ဝဲလ် မီနူး</string>
    </property>
   </widget>
   <widget class="QPushButton" name="inventoryBtn">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>159</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>ဂိုလ်ဒေါင်ထဲကြည့်မယ်။</string>
    </property>
   </widget>
   <widget class="QPushButton" name="salePosBtn">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>159</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>စျေးရောင်းမယ်။</string>
    </property>
   </widget>
   <widget class="QPushButton" name="productsBtn">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>130</y>
      <width>159</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>ကုန်ပစ္စည်းများ</string>
    </property>
   </widget>
   <widget class="QPushButton" name="saleReportBtn">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>170</y>
      <width>159</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>အရောင်းစာရင်း</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>130</y>
      <width>256</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="addItemBtn">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>230</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>addItems</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
